etk/unit_tests/test_DBPediaSpotlight_extractor.py

```diff
@@ -14,7 +13,9 @@ def test_DBPediaSpotlight_extractor(self) -> None:
         confidence = 0.5
 
         extractor = DBpediaSpotlightExtractor(extractor_name='dbPedia_extractor',
-                                              search_url='http://model.dbpedia-spotlight.org/en/annotate')
+                                              search_url='http://model.dbpedia-spotlight.org/en/annotate',
+                                              get_attr=False,
```

Comment:
There is no unit test for get_attr=True, please add atleast one

Reply:
If I set get_attr=True, the result would be huge, maybe hundreds or thousands lines for just one phrase. Because attributes contains lots of wiki links and paragraphs. It can be hard to code into python files.